packages/cojson-storage-indexeddb/src/idbClient.ts

```diff
@@ -220,7 +249,15 @@ export class IDBClient implements DBClientInterface {
     );
   }
 
-  async unitOfWork(operationsCallback: () => unknown[]) {
-    return Promise.all(operationsCallback());
+  async transaction(operationsCallback: () => unknown) {
```

Comment:
Simplified the signature here to make it easier to use when the intent is to have a transaction with rollback